README.adoc

```diff
@@ -2,7 +2,6 @@
 ifdef::env-github,env-browser[:relfileprefix: docs/]
 
 https://travis-ci.org/CS2103-AY1819S1-T12-4/main[image:https://travis-ci.org/CS2103-AY1819S1-T12-4/main.svg?branch=master[Build Status]]
-https://coveralls.io/github/CS2103-AY1819S1-T12-4/main?branch=master[image:https://coveralls.io/repos/github/CS2103-AY1819S1-T12-4/main/badge.svg?branch=master[Coverage Status]]
```

Comment:
Not sure if we should be removing this?

Reply:
actually dont see the badge on our readme ><

Reply:
I will add back.

Reply:
I add this badge yesterday, it fail my PR so I close it. Will open it again.